.gitignore

```diff
@@ -12,12 +12,4 @@
 
 # Package Files #
 *.jar
-*.war
-*.nar
-*.ear
 *.zip
-*.tar.gz
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
```

Comment:
Still not looking like [our standard .gitignore for maven projects](https://github.com/companieshouse/styleguides/blob/master/gitignore/maven)

Reply:
Hopefully fixed in next PR